Makefile

```diff
@@ -19,7 +19,14 @@ BRANCH := latest
 
 PROJECT_BASE := github.com/skycoin/skywire
 DMSG_BASE := github.com/skycoin/dmsg
-OPTS?=GO111MODULE=on
+ifeq ($(OS),Windows_NT)
+	OPTS?=powershell -Command setx GO111MODULE on;
+	DATE := $(shell powershell -Command date -u ${RFC_3339})
+
+else
+	OPTS?=GO111MODULE=on
```

Comment:
We need to use `DATE` inside the else too.

Reply:
there's already a DATE variable defined above, so no need to define it twice

Reply:
Yes, what I meant was that DATE variable should be inside the else statement as `make` isn't working in windows if it is outside.